generators/generator-kotlin-constants.js

```diff
@@ -17,13 +17,19 @@
  * limitations under the License.
  */
 
-const KOTLIN_VERSION = '1.4.21';
+// TODO: Upgrade tp 1.5+
```

Comment:
I would prefer creating an issue here and tracking rather than adding a TODO?

Reply:
Sure thing.